### ISSUE TITLE
Polaris Sample Changer: 

### ISSUE THREAD

**Issue by Alistair-McGann-Tessella** [ISISComputingGroup/IBEX#4760]
As a user I don't want to accidentally initialise my sample changer. I have noticed that when I enter a position number in the OPI and press enter, the focus moves to the initialise button. If enter was pressed more than once, this would trigger an unwanted initialisation.

### Acceptance criteria
- [ ] OPI focus does not move to 'initialise' after typing in a position number and pressing 'enter'

**Comment by DominicOram** [comment]
Can I suggest we do #4786 instead?

**Comment by DominicOram** [comment]
Check this is fixed by #4786

**Comment by Tom-Willemsen** [comment]
No, it still focuses back onto the initialize button on master.
